MonitorControl/Info.plist

```diff
@@ -28,9 +28,13 @@
 	<true/>
 	<key>NSHumanReadableCopyright</key>
 	<string>MIT Licensed. 2017.</string>
-	<key>NSMainNibFile</key>
-	<string>MainMenu</string>
 	<key>NSPrincipalClass</key>
 	<string>NSApplication</string>
+	<key>SUEnableAutomaticChecks</key>
+	<true/>
+	<key>SUFeedURL</key>
+	<string>https://monitorcontrol.app/betachannel.xml</string>
+	<key>SUPublicEDKey</key>
+	<string>ITSTMp8AypsLawojJ+UR3tm2mN18AFoNMvXf1G3t62s=</string>
```

Comment:
Will look more into the whole Sparkle config tomorrow too, would be great if we can get beta distribution going through Sparkle out of the gate.

Reply:
I agree. But as @the0neyouseek mentioned we don't need separate xml-s for the beta and main release so it might as they can contain both and there could be a preference about which one the user chooses to subscribe to.

I changed the `SUFeedURL` value now to `$(APPCAST_ENDPOINT)` but please feel free to modify it in any way you fit as you delve deeper into Sparkle!